src/clarity-angular/data/datagrid/built-in/filters/datagrid-string-filter.ts

```diff
@@ -68,7 +68,7 @@ export class DatagridStringFilter extends DatagridFilterRegistrar<DatagridString
                 // We need the timeout because at the time this executes, the input isn't
                 // displayed yet.
                 setTimeout(() => {
-                    this.renderer.invokeElementMethod(this.input.nativeElement, "focus");
+                    this.input.nativeElement.focus();
```

Comment:
Ok, I'm going to ask you to do a small change even though what you wrote here is the exact same behavior as we had before. The thing is that at the time we wrote the original code, `renderer.invokeElementMethod()` was supposed to work in webworkers. I'm aware that the Angular Renderer has changed since then and now fails when calling this method in a webworker.

We want to try a open the possibility, in the future, to have the datagrid work in webworker, with server-side rendering, or in any other headless mode. To do this, we made sure that every access to native elements API for the datagrid is in our own [DomAdapter](https://github.com/vmware/clarity/blob/master/src/clarity-angular/data/datagrid/render/dom-adapter.ts). So I would like you to write a one-line method in it:
```typescript
focus(element: any) {
   element.focus();
}
```
And just call this method here. Yes, it seems extreme, but that way we group all our not-headless-compatible code in a single class, in which we could add guards, tests or even a separate implementation for server-side rendering.